src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -187,26 +184,10 @@ public void parseYear_validYear_returnsYear() throws Exception {
     @Test
     public void parseYear_validYearWithWhiteSpace_returnsTrimmedYear() throws Exception {
         String yearWithWhiteSpace = WHITESPACE + VALID_YEAR + WHITESPACE;
-        Year expectedYear = new Year(VALID_SCHOOL_TYPE, VALID_SCHOOL_LEVEL);
+        Year expectedYear = new Year(VALID_YEAR);
         assertEquals(expectedYear, ParserUtil.parseYear(yearWithWhiteSpace));
     }
 
-    @Test
-    public void parseSchoolType_validSchoolType_returnsCorrectSchoolType() throws ParseException {
-        assertEquals(SchoolType.PRIMARY, ParserUtil.parseSchoolType("Primary")); // full word first letter capitalised
```

Comment:
I think can consider adapting these tests and moving over to the `YearTest.java` instead of deleting if the parsing of SchoolType is moved to `Year.java`? Because right now `YearTest` does not have much tests as the constructor assigns only the given `schoolType` and `level` to attributes but if the parsing is moved over there I think can potentially add more tests there.

Reply:
Good idea, I forgot about it :P